cmd/main.go

```diff
@@ -470,5 +429,9 @@ func (ss *StringSlice) Set(value string) error {
 	return nil
 }
 
+// TODO: should it implement either of those?
+// var _ pflag.Value = (*StringSlice)(nil)
+// var _ pflag.SliceValue = (*StringSlice)(nil)
```

Comment:
Do we need to do anything with this?

Reply:
I'd say let's wait on that unless we really have a need for it